### ISSUE TITLE
App token is generating when schedule script is called

### ISSUE THREAD

**Issue by kodifan** [ironmansoftware/powershell-universal#3271]
### Version

4.2.13

### Severity

Low

### Environment

msi

### Steps to Reproduce

I have a schedule that every hour is updating some variables from active directory. 
Its running for quite some time already and its working fine.
Recently i noticed a lot (like really a lot - almost 1000) app tokens 
The time when those tokens were generated match time when schedule is triggered. 
Since those tokens have one year live time to expiration  they are keeping growing constantly. 
I don't know whatever it affects PU performance in any way (maybe database size increase )   but i guess it should not be this way

Is there any way to shorten the time which tokens are valid to one day or so ?

### Expected behavior

```console
app token generated when schedule script is called have short expiration time.
```

### Actual behavior

```console
app token generated when schedule script is called have one year expiration time.
```

### Screenshots/Animations

![image](https://github.com/ironmansoftware/issues/assets/7819312/3252c791-b5a9-4800-b879-5537cfb206e8)

**Comment by kodifan** [comment]
ok looks like resolved in 4.2.14 , time to update
btw i managed to revoke all those 900 tokens using PU api, but how to actually delete them ?

**Comment by adamdriscoll** [comment]
You'd have to do so manually in the database. They will automatically be groomed by the system after 30 days
